ui/app/common/displaycontrols/tabularview/directives/obsToObsFlowSheet.js

```diff
@@ -23,6 +23,15 @@ angular.module('bahmni.common.displaycontrol.obsVsObsFlowSheet').directive('obsT
                 });
             };
 
+            const removeEmptyRecords = function (records) {
```

Comment:
personally I think it's bad style, and can be confusing for the caller, if you both mutate the input _and_ also return it. (But I will still merge anyway.)